codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/SourceGenerator.scala

```diff
@@ -57,35 +85,26 @@ object SourceGenerator {
    * user.
    */
   def generateUnmanaged(model: ModelBuilder.Model): Seq[File] = {
-    Seq(File("foo/bar/Baz.scala", "package foo.bar\n\nclass Baz extends AbstractBaz"), generateMain(model)) ++
-    model.services.values
+    val filteredModel = excludedUntilImplemented(model) // FIXME remove
+    Seq(File("foo/bar/Baz.scala", "package foo.bar\n\nclass Baz extends AbstractBaz")) ++
+    MainSourceGenerator.generateUnmanaged(filteredModel) ++
+    filteredModel.services.values
```

Comment:
Wouldn't the fact that we use the `filteredModel` here mean we're no longer generating the `ValueEntitySourceGenerator.generateImplementationSkeleton` while we did before?
(my fault for not checking for it in the scripted tests I guess ;) )

Reply:
ok, I moved this filteredModel to MainSourceGenerator instead